test/sdk-benchmarks/src/main/resources/baseline_score.json

```diff
@@ -0,0 +1,22 @@
+[
```

Comment:
Yeah, not sure how we can have two versions in the code. You would need to firstly run the version before the change and then run it again with the new change and compare the results.  

I think the best way to do it is using the same Codebuild job and compare it with the baseline.

Reply:
One way to do it would be to not shade the tested classes into `benchmarks.jar` so we could swap out different JAR versions and rerun.

Having this flexibility would allow someone who doesn't have access to the same hardware that baseline was run against to more easily check for perf improvements or regressions.